tests/shouldwork/Xilinx/ClockWizard.hs

```diff
@@ -19,15 +19,15 @@ createDomain vXilinxSystem{vName="DomOut", vPeriod=hzToPeriod 300_000_000}
 
 topEntity ::
   Clock DomIn ->
-  Clock DomIn ->
+  DiffClock DomIn ->
```

Comment:
I don't understand this change, how can a _single_ clock change to a `DiffClock`?

Reply:
This test isn't really suitable to be synthesised, I'm pretty cavalier with the domains and `unsafeSynchronizer`. The first input was the non-inverted clock and the second input the inverted clock, and the non-inverted clock was fed to a clock wizard with a single-ended input as well. This test tests that both `clockWizard` and `clockWizardDifferential` assert their locked output and make actual progress. So with the new setup, we need a single-ended clock and a differential clock, where previously we could just feed the non-inverted signal to both.